src/store/getters.ts

```diff
@@ -1147,9 +1116,10 @@ export const getFolioNumber = (state: StateIF): string => {
   return state.stateModel.newRequestModel.folioNumber
 }
 
-/** Name Check Getters
- * TODO: move existing getters used only for name check above to here
- * TODO: eventually move this all out of vuex (if we refactor to composition api)
```

Comment:
I went through the code and implemented/deleted my TODOs (and changed the others to FUTUREs).